react/features/video-menu/components/AbstractMuteEveryonesVideoDialog.js

```diff
@@ -29,12 +38,36 @@ export type Props = AbstractProps & {
  *
  * @extends AbstractMuteRemoteParticipantsVideoDialog
  */
-export default class AbstractMuteEveryonesVideoDialog<P: Props> extends AbstractMuteRemoteParticipantsVideoDialog<P> {
+export default class AbstractMuteEveryonesVideoDialog<P: Props>
+    extends AbstractMuteRemoteParticipantsVideoDialog<P, State> {
     static defaultProps = {
         exclude: [],
         muteLocal: false
     };
 
+    /**
+     * Initializes a new {@code AbstractMuteRemoteParticipantsVideoDialog} instance.
+     *
+     * @param {Object} props - The read-only properties with which the new
+     * instance is to be initialized.
+     */
+    constructor(props: P) {
+        super(props);
+
+        this.state = {
+            moderationEnabled: props.isVideoModerationEnabled,
+            content: props.content || props.t(
+                `dialog.muteEveryonesVideoDialog${props.isVideoModerationEnabled
+                    ? 'ModerationOn'
+                    : ''}`
```

Comment:
Can you use the full strings, here please? This causes problems in the future when searching whether a string is used in the code, thanks.